tutorials/assets_pipeline/importing_scenes.rst

```diff
@@ -29,6 +29,33 @@ Copy the scene file together with the textures and mesh data (if separate) to
 the project repository, then Godot will do a full import when focusing the
 editor window.
 
+3D asset direction conventions
+------------------------------
+
+Godot uses a right-handed, Y-is-up coordinate system, with the -Z axis as
+the camera's forward direction. This is the same as OpenGL. This implies
+that +Z is back, +X is right, and -X is left for a camera.
+
+The convention for 3D assets is to face the opposite direction as the camera,
```

Comment:
It is said as a convention in here, but my understanding is that the definition of the right-hand coordinate system is that the camera and the asset are facing each other as the assumption. But I am not certain that this is the truth, so it may be best to defer to reduz for judgment.

Reply:
The handedness of the coordinate system itself does not make any assumptions of what things exist in it. The handedness only determines how the axes relate to each other. That is, if you know X and Y, where is Z? In right-handed, +Y up and +X right implies +Z back, alternatively +Y up and +X left implies +Z forward, etc.

Reply:
What I am saying is that in "any" coordinate system, the +X axis of the camera is consistent with being on the right.

Reply:
That's not how handedness is defined mathematically, though. Math doesn't know about assets or cameras.

+X camera right is not consistent in all apps thanks to Unreal throwing a monkey wrench into things. Unreal uses +X forward, +Y right, and +Z up (this is left-handed).